testsuite/pytests/test_iaf_ps_psp_accuracy.py

```diff
@@ -20,14 +20,19 @@
 # along with NEST.  If not, see <http://www.gnu.org/licenses/>.
 
 """
-test_iaf_ps_psp_accuracy.sli checks the voltage response of the precise versions of the iaf_psc_alpha
-model neurons to a single incoming spike. The voltage excursion is
-called postsynaptic potential (PSP). In the model neurons
-the postsynaptic current is described by an alpha-function
+Compares the voltage response of iaf_psc_alpha_ps from a single incoming
+excitatory spike at an off-grid time to the analytical solution. 
+Also checks that the equivalent non-ps iaf_psc_alpha does not give the same result.
```

Comment:
Start with a single line which could be included in a table of tests, e.g.,
```suggestion
Test that neuron with precise timing has correct post-synaptic potential for off-grid input.

Compares the voltage response of iaf_psc_alpha_ps from a single incoming
excitatory spike at an off-grid time to the analytical solution. 
Also checks that the equivalent non-ps iaf_psc_alpha does not give the same result.
```

Reply:
Done